services/src/main/java/org/keycloak/services/resources/account/AccountFormService.java

```diff
@@ -982,9 +982,8 @@ public static boolean isPasswordSet(KeycloakSession session, RealmModel realm, U
                 return new String[]{referrerName, referrerUri};
             }
         } else if (referrerUri != null) {
-            referrerClient = realm.getClientByClientId(referrer);
```

Comment:
This line is unnecessary because there is the same line (L969) and value is not changed.